os/board/common/crashdump.c

```diff
@@ -450,6 +511,12 @@ static int dump_via_uart(void)
 			lldbg("Unable to reset device\n");
 #endif
 			break;
+		/* Check handshake string for USERFS FILE DUMP signal */
+		} else if (strncmp(host_buf, USERFS_FDUMP_HANDSHAKE_STRING, strlen(USERFS_FDUMP_HANDSHAKE_STRING)) == 0) {
```

Comment:
Kinldy use strnlen()

Reply:
All handshake checks are currently using `strlen`, so sticking with the same.